lib/tasks/cait_indc.rake

```diff
@@ -0,0 +1,6 @@
+namespace :cait_indc do
+  desc 'Imports the CAIT INDC dataset from the csv sources'
+  task import: :environment do
```

Comment:
in this PR https://github.com/Vizzuality/climate-watch/pull/43 there's a "import them all" rake task which I think is going to be handy as order of import starts to matter, just letting you know in case you were contemplating adding sth like that :)

Reply:
yes, thanks for the heads up! 👍